### ISSUE TITLE
Accessibility: 

### ISSUE THREAD

**Issue by zeelyha** [jupyterhub/outreachy#60]
### What page is this for?

https://jupyter.org/governance/conduct/reporting_events.html

### WAVE accessibility report

![Screenshot (21)](https://user-images.githubusercontent.com/73789529/195985398-0e92c06b-7bc8-4eb1-8d7a-56667ffee963.png)

-  Missing language attribute: The language of the document is not specified.
- Redundant link: Adjacent links have the same destination URL.
- Suspicious link: A link text is not clearly worded.
- Empty button: A button on the page has no value text.

### The HTML element to be changed

```html
1. <html>



2. Email:
Redundant link
<a class="reference external" href="mailto:conduct%40jupyter.org">
<em>
conduct
<span>
@
</span>

jupyter
<span>
.
</span>

org
</em>
</a>

3. You also have the option of following the reporting procedure for the online community found
<a class="reference external" href="https://goo.gl/forms/sJzOIie3zde9M71T2">
here
</a>


4. <button onclick="toggleFullScreen()" class="headerbtn" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="Fullscreen mode">
<span class="headerbtn__icon-container">
<i class="fas fa-expand"></i>
</span>
</button>
```

### Your proposed HTML (or CSS), after change

```html
1. <html lang = "en">

2. <a class="reference external" href="mailto:conduct%40jupyter.org">conduct@jupyter.org</a>


3. You also have the option of following the reporting procedure for the online community found
<a class="reference external" href="https://goo.gl/forms/sJzOIie3zde9M71T2">
in the code of conduct.
</a>


4. <button onclick="toggleFullScreen()" class="headerbtn" value="fullscreen-toggle" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="Fullscreen mode">
<span class="headerbtn__icon-container">
<i class="fas fa-expand"></i>
</span>
</button>
```

**Comment by minrk** [comment]
Thank you! Can you discuss why removing styling with span/em in the email link fixes an accessibility issue?

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
